sdk/storage/azure-storage-file-datalake/tests/test_file_system_async.py

```diff
@@ -235,52 +236,18 @@ async def test_undelete_file_system(self, datalake_storage_account_name, datalak
             filesystem_list.append(fs)
         self.assertTrue(len(filesystem_list) >= 1)
 
-        restored_version = 0
         for filesystem in filesystem_list:
             # find the deleted filesystem and restore it
             if filesystem.deleted and filesystem.name == filesystem_client.file_system_name:
-                restored_fs_client = await self.dsc.undelete_file_system(filesystem.name,
-                                                                         filesystem.deleted_version,
-                                                                         new_name="restored" +
-                                                                                  name + str(restored_version))
-                restored_version += 1
+                restored_fs_client = await self.dsc.undelete_file_system(filesystem.name, filesystem.deleted_version)
 
                 # to make sure the deleted filesystem is restored
                 props = await restored_fs_client.get_file_system_properties()
                 self.assertIsNotNone(props)
 
-    @DataLakePreparer()
-    async def test_restore_to_existing_file_system(self, datalake_storage_account_name, datalake_storage_account_key):
-        # Needs soft delete enabled account.
-        if not self.is_playback():
-            return
-        self._setUp(datalake_storage_account_name, datalake_storage_account_key)
-        # get an existing filesystem
-        existing_name = self._get_file_system_reference(prefix="existing")
-        name = self._get_file_system_reference(prefix="filesystem")
-        existing_filesystem_client = await self.dsc.create_file_system(existing_name)
-        filesystem_client = await self.dsc.create_file_system(name)
-
-        # Act
-        await filesystem_client.delete_file_system()
-        # to make sure the filesystem deleted
-        with self.assertRaises(ResourceNotFoundError):
-            await filesystem_client.get_file_system_properties()
-
-        filesystem_list = []
-        async for fs in self.dsc.list_file_systems(include_deleted=True):
-            filesystem_list.append(fs)
-        self.assertTrue(len(filesystem_list) >= 1)
-
-        for filesystem in filesystem_list:
-            # find the deleted filesystem and restore it
-            if filesystem.deleted and filesystem.name == filesystem_client.file_system_name:
-                with self.assertRaises(HttpResponseError):
-                    await self.dsc.undelete_file_system(filesystem.name, filesystem.deleted_version,
-                                                        new_name=existing_filesystem_client.file_system_name)
-
```

Comment:
I think we still need to test restoring to existing filesystem/container, while it can be added later and tracked in an issue
the steps are
1. use soft delete enabled account
2. create container A and soft delete it
3. recreate container A
4. restore deleted container and see if we can get HttpResponseError